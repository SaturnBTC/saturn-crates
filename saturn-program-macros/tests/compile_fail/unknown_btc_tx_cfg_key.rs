use saturn_program_macros::saturn_program;

mod instruction {
    #[derive(borsh::BorshSerialize, borsh::BorshDeserialize)]
    pub enum Instr {
        Dummy,
    }
}

#[saturn_program(
    btc_tx_cfg(foo = 1, max_inputs_to_sign = 1, max_modified_accounts = 1)
)]
mod handlers {}

fn main() {} 