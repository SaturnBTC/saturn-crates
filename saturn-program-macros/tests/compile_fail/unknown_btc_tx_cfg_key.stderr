error: unknown key inside btc_tx_cfg; expected `max_inputs_to_sign`, `max_modified_accounts`, or `rune_set`
  --> tests/compile_fail/unknown_btc_tx_cfg_key.rs:13:16
   |
13 |     btc_tx_cfg(foo = 1, max_inputs_to_sign = 1, max_modified_accounts = 1)
   |                ^^^^^^^
